### PR TITLE
[TASK] Clean up code base

### PR DESCRIPTION
**Pull request by bmack**
- Use FQDN for use statements
- Fix some variable names and phpdoc info (typos mostly)
- Remove unused functionality
- Simplify code (e.g. notices via PHPunit directly)
- Use native PHP functionality over Core APIs when useful

### CODE REVIEW

File: Classes/Core/Functional/Framework/Frontend/Response.php [hunk 1]
```diff
@@ -19,7 +19,6 @@
  */
 class Response
 {
-    const STATUS_Success = 'success';
```
Comment (ohader) [inline Classes/Core/Functional/Framework/Frontend/Response.php]: Not sure about changes in the `Response` layer... probably currently not used, but might be useful in future™

File: Classes/Core/Functional/Framework/Frontend/ResponseContent.php [hunk 2]
```diff
@@ -108,14 +103,6 @@ function (string $sectionIdentifier) {
         );
     }
 
-    /**
-     * @return array
-     */
-    public function getScope(): array
```
Comment (ohader) [inline Classes/Core/Functional/Framework/Frontend/ResponseContent.php]: Not sure about changes in the `Response` layer... probably currently not used, but might be useful in future™

Comment (ohader) [reply Classes/Core/Functional/Framework/Frontend/ResponseContent.php]: Adjust restrictions like
```
$queryBuilder->getRestrictions()->removeAll()->add(GeneralUtility::makeInstance(DeletedRestriction::class));```

Comment (bmack) [reply Classes/Core/Functional/Framework/Frontend/ResponseContent.php]: We can re-add later. You know... Git...

File: Classes/Core/Functional/Framework/Frontend/Response.php (hunk 1 above)
Comment (bmack) [reply Classes/Core/Functional/Framework/Frontend/Response.php]: We could/should use status codes instead, once we need it

**Pull request merged**